src/main/java/com/powsybl/openloadflow/network/impl/LfShuntImpl.java

```diff
@@ -140,6 +140,11 @@ public String getId() {
         return shuntCompensators.stream().map(ShuntCompensator::getId).collect(Collectors.toList());
```

Comment:
Can we delete this `getIds` method then?

Reply:
Done